runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -116,18 +116,29 @@ default boolean sendNotificationsWhenFocused()
 		keyName = "notificationRequestFocus",
 		name = "Request focus on notification",
 		description = "Toggles window focus request",
-		position = 8
+		position = 13
 	)
 	default boolean requestFocusOnNotification()
 	{
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "customNotifications",
+		name = "Use non-native notifications",
+		description = "Use RuneLite styled notifications",
+		position = 14
+	)
+	default boolean customNotifications()
+	{
+		return System.getProperty("os.name").toLowerCase().contains("win");
```

Comment:
I would make this disabled by default, this is problem only on Windows 10

Reply:
Windows 8+ has this problem

Reply:
Still. Also, not everyone complained about this, I would just make this opt-in every time, as native system functionality should be always preferred.

Reply:
Native functionally is FUBAR for most users.

Reply:
The problem has to do with W10 toasts (the shitty replacement for balloon notifications) have dead zones. You just have to either make a regedit or download a program to revert them and you won't have this issue.